<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowWire</class>
 <widget class="QDialog" name="WindowWire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>531</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>531</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Helena - Fios</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/main.qrc">
    <normaloff>:/iconPower.ico</normaloff>:/iconPower.ico</iconset>
  </property>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>165</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_length">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_type">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_area">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_weight">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Peso (kg/km):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_weight">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_length">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>165</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comprimento (m/kg):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_turnPerCm">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>165</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Espiras por centrimetro (esp/cm):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_insert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserir Novo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_before">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_frequency">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_area">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Area (mm^2):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_diameter">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Diametro (mm):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_awg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>165</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AWG:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_awg">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Atualizar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_after">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_resistance">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_resistance">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>165</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Resistencia (Ω/km):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_diameter">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>185</y>
     <width>511</width>
     <height>168</height>
    </rect>
   </property>
   <property name="title">
    <string>Corrente Maxima</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_frequency">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequencia (kHz):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_turnsPerCm">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_material">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>165</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Material:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_material">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_currentMax">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>491</width>
     <height>131</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>densidade corrente (A/mm^2)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>corrente máxima (A)</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit_provider">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_provider">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fornecedor:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Excluir</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_type</tabstop>
  <tabstop>lineEdit_awg</tabstop>
  <tabstop>lineEdit_diameter</tabstop>
  <tabstop>lineEdit_turnsPerCm</tabstop>
  <tabstop>lineEdit_area</tabstop>
  <tabstop>lineEdit_resistance</tabstop>
  <tabstop>lineEdit_weight</tabstop>
  <tabstop>lineEdit_length</tabstop>
  <tabstop>lineEdit_frequency</tabstop>
  <tabstop>lineEdit_material</tabstop>
  <tabstop>tableWidget_currentMax</tabstop>
  <tabstop>pushButton_before</tabstop>
  <tabstop>pushButton_after</tabstop>
  <tabstop>pushButton_update</tabstop>
  <tabstop>pushButton_insert</tabstop>
  <tabstop>pushButton_exit</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/main.qrc"/>
 </resources>
 <connections/>
</ui>
